adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -473,6 +507,25 @@ def connect_AP(self, ssid, password): # pylint: disable=invalid-name
             raise RuntimeError("No such ssid", ssid)
         raise RuntimeError("Unknown error 0x%02X" % stat)
 
+    def create_ap(self, ssid, password, channel=b'\x01'):
+        """Create an access point with the given name and password."""
+        if isinstance(ssid, str):
```

Comment:
Perform the this check within `set_ap_passphrase` instead, along with checking the length of the ssid.

Reply:
since `_wifi_set_ap_passphrase` and `_wifi_set_ap_network` are now both listed as private, I think it makes more sense to do all validation (including ssid/passphrase length checking and channel being 1-14) and arg documentation in `create_AP`, since that's the public method to be used.

This is also similar to the way `connect_AP` is written to provide a main entry point for calling either `wifi_set_passphrase` and `wifi_set_network`

Reply:
I agree this makes more sense here. This is the function that the user interacts with and it makes sense that it holds the business  rules about error messages, etc.